superset/config.py

```diff
@@ -370,6 +370,7 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
     "en": {"flag": "us", "name": "English"},
     "es": {"flag": "es", "name": "Spanish"},
     "it": {"flag": "it", "name": "Italian"},
+    "fa": {"flag": "ir", "name": "Persian"},
```

Comment:
It's good to add this here, but line 389 below disables this whole block. For testing purposes, you can add a few languages there and they should show up on the ephemeral/test environments on this PR.

Sorry I haven't had time to build/test this locally yet... I'm still hoping to.

Reply:
Done. You're right, I didn't check other lines. 
Don't even mention it. Thanks for helping me out.